### ISSUE TITLE
Cannot compile after upgrading react-native

### ISSUE THREAD

**Issue by yuameshi** [pchmn/expo-material3-theme#4]
```
$ npm run build:android:release

> x@0.0.1 build:android:release
> cd android/ && ./gradlew assembleRelease && ./gradlew bundleRelease

Downloading https://services.gradle.org/distributions/gradle-8.0.1-all.zip
...............10%................20%................30%................40%................50%................60%................70%................80%................90%................100%

Welcome to Gradle 8.0.1!

Here are the highlights of this release:
 - Improvements to the Kotlin DSL
 - Fine-grained parallelism from the first build with configuration cache
 - Configurable Gradle user home cache cleanup

For more details see https://docs.gradle.org/8.0.1/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* Where:
Build file '/workspaces/*/node_modules/@pchmn/expo-material3-theme/android/build.gradle' line: 40

* What went wrong:
A problem occurred evaluating project ':pchmn-expo-material3-theme'.
> Could not set unknown property 'classifier' for task ':pchmn-expo-material3-theme:androidSourcesJar' of type org.gradle.api.tasks.bundling.Jar.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
A problem occurred configuring project ':expo'.
> compileSdkVersion is not specified. Please add it to build.gradle

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 35s
5 actionable tasks: 5 executed
```
# DIFF of `settings.gradle`
```diff
rootProject.name = 'x'
apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
include ':app'
- includeBuild('../node_modules/react-native-gradle-plugin')
+ includeBuild('../node_modules/@react-native/gradle-plugin')
apply from: new File(["node", "--print", "require.resolve('expo/package.json')"].execute(null, rootDir).text.trim(), "../scripts/autolinking.gradle")
useExpoModules()
```

**Comment by yuameshi** [comment]
find a workaround at [here](https://github.com/expo/expo/issues/23023)

**Comment by yuameshi** [comment]
UPDATE: fixed after using gradle 7.5.1 instead of 8.0.1

### `android/gradle/wrapper/gradle-wrapper.properties`
```diff
- distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.1-all.zip
+ distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
```

**Comment by pchmn** [comment]
I'll close this issue if you don't mind, because it's a compatibility problem between expo and gradle version 8, and it will be fixed in next expo versions

**Comment by pchmn** [comment]
@yuameshi I just published a new version ([v1.1.0](https://github.com/pchmn/expo-material3-theme/releases/tag/v1.1.0)) to fix issue with gradle 8, you can try it if you want

**Comment by yuameshi** [comment]
@pchmn i'll try it tomorrow, thanks for your working.
